.circleci/config.yml

```diff
@@ -31,17 +31,24 @@ parameters:
   cache_version:
     type: string
     # increment that one to invalidate all the caches
-    default: v8.{{ checksum "rust-toolchain.toml" }}
+    default: gary.{{ checksum "rust-toolchain.toml" }}
```

Comment:
note to us, make sure it s removed once we mark it as ready to review

Reply:
I've reversed it out. I couldn't spot a worthwhile improvement.